From 7d6e02f6696e93b2e57225a8adfa1a1d89d633ed Mon Sep 17 00:00:00 2001
From: Lawrence-Tang <tangzongsheng@gmail.com>
Date: Sun, 23 Feb 2020 16:43:34 +0800
Subject: [PATCH 24/55] feeds: update to the latest commit

---
 feeds.conf.default | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/feeds.conf.default b/feeds.conf.default
index 8ccb794398..8c4dbb9766 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -1,5 +1,5 @@
 src-git packages https://github.com/friendlyarm/friendlywrt_project_packages^5b22f3f59c03412200fe4815d833943f953edf2b
-src-git luci https://github.com/friendlyarm/friendlywrt_project_luci^df0e22b5294fdf8e8964e810d6834d00f65fc246
+src-git luci https://github.com/friendlyarm/friendlywrt_project_luci^81580a3307e5a723ec4c01783de24e8e28c8cbd8
 src-git routing https://git.openwrt.org/feed/routing.git^f1b0476edc5486793e098fe83089054bb2941663
 src-git telephony https://git.openwrt.org/feed/telephony.git^6f95d6ab3f359ee2ce81c20522700937424d1591
 #src-git video https://github.com/openwrt/video.git
-- 
2.14.1.windows.1

