From 9f2244ea0da0a73cc8183fadd0c664aaf96ada18 Mon Sep 17 00:00:00 2001
From: Chuck <fanck0605@qq.com>
Date: Mon, 11 May 2020 18:18:06 +0800
Subject: [PATCH 30/39] rk3328: adapt to rockchip-kernel's HZ settings

---
 target/linux/rockchip-rk3328/config-4.14 | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/target/linux/rockchip-rk3328/config-4.14 b/target/linux/rockchip-rk3328/config-4.14
index 09d22d517f..5c559ff8e3 100644
--- a/target/linux/rockchip-rk3328/config-4.14
+++ b/target/linux/rockchip-rk3328/config-4.14
@@ -311,6 +311,9 @@ CONFIG_HUGETLB_PAGE=y
 CONFIG_HWSPINLOCK=y
 CONFIG_HW_RANDOM=y
 CONFIG_HW_RANDOM_CAVIUM=y
+CONFIG_HZ=250
+# CONFIG_HZ_100 is not set
+CONFIG_HZ_250=y
 CONFIG_I2C=y
 CONFIG_I2C_ALGOBIT=y
 CONFIG_I2C_BOARDINFO=y
-- 
2.28.0.windows.1

