From 4760a6a19431ecbc0622b15ae65463afd2c9deaa Mon Sep 17 00:00:00 2001
From: kongfl888 K <kongfl888@outlook.com>
Date: Wed, 22 Jul 2020 02:31:34 +0800
Subject: [PATCH 53/55] rk3328: add into supported_devices

---
 target/linux/rockchip-rk3328/image/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/target/linux/rockchip-rk3328/image/Makefile b/target/linux/rockchip-rk3328/image/Makefile
index 203f0ea5c6..2a0836733d 100644
--- a/target/linux/rockchip-rk3328/image/Makefile
+++ b/target/linux/rockchip-rk3328/image/Makefile
@@ -18,4 +18,9 @@ define Image/Build
 	cp $(KDIR)/vmlinux $(BIN_DIR)/$(IMG_PREFIX)-vmlinux
 endef
 
+define Device/Default
+    PROFILES := Default
+    SUPPORTED_DEVICES :=friendlyelec,nanopi-r2
+endef
+
 $(eval $(call BuildImage))
-- 
2.14.1.windows.1

